src/main/model/CurlCommand.java

```diff
@@ -0,0 +1,95 @@
+package main.model;
+/* Model for the curl command: \"curl -X 'GET' \
+       'https://financial-integration.service.cnqr.tech/fi/v1/companies/1/transactiontypes/expense/transactions?limit=0&docId=%s&ignoreDocumentStatus=yes' \
+       -H 'accept: application/json' \
+       -H 'documentFormatAs: json' \
+      -H 'Authorization: Bearer $JWT'\" \
+       > ~/Downloads/%s.json;
+*/
+public class CurlCommand {
+    private String method;
+    private String url;
+    private String ignoreDocStatus;
```

Comment:
The name of this variable says it should be a boolean. I see it needs to be written as "yes" and not "true", but you can write a converter for that; this is best as a boolean. :)